### DISCUSSION TITLE
Welcome to doubleml-for-r Discussions!

### DISCUSSION THREAD

**Discussion by PhilippBach** [DoubleML/doubleml-for-r#144] [General]
## 👋 Welcome to DoubleML Discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas :bulb:.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by GeorBe** [comment]
Is it possible to extract nuisance function estimates and other intermediate results from IRM fit object? I need estimates of the probability of receiving the treatment and also if possible which of the covariates were retained, rather than simply getting the final estimate.
